recipes/category_encoders/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set version = "1.2.1" %}
+{% set gh_org = "wdm0006" %}
+{% set gh_repo = "categorical_encoding" %}
+{% set pkg_name = "category_encoders" %}
+
+
+package:
+  name: {{ pkg_name }}
+  version: {{ version }}
+
+source:
+  fn: {{ gh_repo }}-{{ version }}.tar.gz
+  url: https://github.com/{{ gh_org }}/{{ gh_repo }}/archive/{{ version }}.tar.gz
+  sha256: d4649bf425c4398c9ed54e5d4021181d652e05189242d823658e6fad9b986c2f
+
+build:
+  number: 0
+  script:
+    # FIXME: This is a hack to make sure the environment is activated.
+    # The reason this is required is due to the conda-build issue
+    # mentioned below.
+    #
+    # https://github.com/conda/conda-build/issues/910
+    #
+    - source activate "${CONDA_DEFAULT_ENV}"  # [unix]
```

Comment:
So, we mainly are using this when we use the `toolchain` as it requires environment activation to work correctly.